### PR TITLE
updated repo structure

### PR DESCRIPTION
**Pull request by justin0022**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,58 @@
...
+Please register for the hackathon if you haven't ready done so: https://events.ctlt.ubc.ca/events/2023-fall-learning-analytics-hackathon/
 
-* a data dictionary
-* a data sample (full data is available to registrants through the Canvas course)
-  
-* a final_product folder
-  * add your final product description and reflection 
-  * add your presentation files or a link to your presentation - suggested areas to cover in `slides.pptx` (use this as a template, a guide, or just inspiration - it is not a requirement to answer all questions)
+## 📚 What You'll Learn
+By completing this workshopathon, you'll learn to:
```
Comment (alisonmyers) [inline README.md]: should we change this to something like "additional resources / info"? / at least remove reference to workshopathon

**Review by alisonmyers — APPROVED** [review]
see comment - otherwise lgtm

File: README.md (hunk 1 above)
Comment (justin0022) [reply README.md]: Oops, fixed. I also moved this section down.

Comment (justin0022) [reply README.md]: I'll merge once @craigdsthompson has a chance to review on Monday.

**Pull request merged**